### ISSUE TITLE
Parting out sets doesn't work

### ISSUE THREAD

**Issue by JWVersluis95** [rgriebl/brickstore#559]
Hi, 

Is there a bug on the moment that recently inventoried sets (expect 10778) can't be parted out in Brickstore? 

Sincerely,
Jan Willem

**Comment by rgriebl** [comment]
Yes there is: the database update was stuck for the last 6 days. I got it unstuck now, so just start a database update and you're back in business.
